locales/en/app.ftl

```diff
@@ -259,3 +261,144 @@ share-twitter = Most people have about 100 online accounts. Have any of yours be
+
+
+mozilla-security-blog = {-brand-Mozilla} Security Blog
+# A header for a list of links to share Firefox Monitor on various social media platforms.
+layout-social = Social
+show-all = Show all
+
+fxa-signup-banner-headline = Monitor threats to your online accounts.
+fxa-signup-banner-blurb = Sign up for a {-brand-fxa} to get your detailed report and alerts about new data breaches that expose your info.
+fxa-landing-blurb = 
+  Find out what hackers already know about you,
+  and learn how to stay a step ahead of them.
+
+fxa-scan-label = See if you’ve appeared in a data breach.
+fxa-welcome-headline = Welcome to {-product-name}.
+fxa-welcome-blurb = You’re all set to get alerts if { $userEmail } appears in a data breach.
+
+fxa-scan-another-email = Want to Check Another Email?
+# Search Firefox Monitor
+fxa-scan-submit = Search {-product-name}
+
+sign-up-to-check = Sign Up to Check
+sign-in = Sign In
+sign-out = Sign Out
+
+full-report-headline = Your {-product-name} Report
+
+see-full-report = See Full Report
+
+# Manage Firefox Account, link to page where account holders can change their account settings.
+manage-fxa = Manage {-brand-fxa}
+
+fxa-download-firefox-bar-blurb = Brought to you by {-brand-name}. 2x faster. Uses 30% less memory than {-brand-Chrome}.
+fxa-download-firefox-bar-link = Download now
+fxa-download-firefox-banner-blurb = Better, faster page loading that uses less computer memory.
+
+sign-up-with-fxa = Sign up for a {-brand-fxa} to get your full report and alerts about new breaches.
+
+compromised-headline = Your account appeared in the { $breachName } data breach.
+no-breaches-found-headline = No breaches found from a basic search.
+
+authenticated-no-breaches-blurb = No breaches found. We’ll send you an email if your info appears in new breaches.
+user-no-breaches-blurb = No breaches found.
+guest-no-breaches-blurb = 
+  For a more in-depth search of this email address in sensitive breaches, 
+  create a {-brand-fxa}. You’ll also get alerts if you appear in new data breaches.
+
+take-action-singular = Take action to protect your account.
+take-action-plural = Take action to protect your accounts.
+
+user-fb-compromised-blurb =
+  { $breachCount ->
+      [0] {take-action-singular}
+      [one] Your email address also appeared in { $breachCount } other breach. {take-action-singular}
+     *[other] Your email address also appeared in { $breachCount } other breaches. {take-action-plural}
+  }
+
+guest-fb-compromised-blurb =
+  { $breachCount ->
+      [0] {sign-up-with-fxa}
+      [one] Your email address also appeared in { $breachCount } other breach. {sign-up-with-fxa}
+     *[other] Your email address also appeared in { $breachCount } other breaches. {sign-up-with-fxa}
+  }
+
+user-fb-not-compromised-blurb =
+  { $breachCount ->
+      [one] You were not in the { $breachName } breach, but we found that email address in another.
```

Comment:
Should this have a `{take-action-singular}` at the end (like the `*[other]` flow)?

Reply:
No, for whatever reason that response is followed by `{take-action-singular}` . https://e-pang.github.io/fm_fxa/#artboard43

Reply:
Any clue why that link goes into a blank page?